src/UI/TransactionsToastList/index.tsx

```diff
@@ -83,19 +83,40 @@ export const TransactionsToastList = ({
   const transactionsToastsList = useMemo(
     () =>
       transactionsToasts.map(
-        ({ toastId, type, startTimestamp }: TransactionToastType) => (
-          <TransactionToast
-            key={toastId}
-            {...{
-              type,
-              startTimestamp,
-              toastId,
-              signedTransactionsToRender,
-              lifetimeAfterSuccess: successfulToastLifetime,
-              onDelete: handleDeleteTransactionToast
-            }}
-          />
-        )
+        ({ toastId, type, startTimestamp }: TransactionToastType) => {
```

Comment:
I think it would be a good idea to pull this out into a separate function outside of the component, renderTransactionToast